.travis.yml

```diff
@@ -15,4 +15,4 @@ notifications:
 
 branches:
   only:
-    - master
+    - swift3
```

Comment:
I think we should keep the `-master` too because this branch will end up there some day